workflow-bot-app/src/test/java/com/symphony/bdk/workflow/api/v1/controller/WorkflowsMgtApiControllerTest.java

```diff
@@ -78,4 +82,11 @@ void test_missingToken_badRequest() throws Exception {
             .content("content"))
         .andExpect(status().isBadRequest());
   }
+
+  @Test
+  void test_streamingLogs_returnOk() throws Exception {
```

Comment:
can we test the error scenario too?

Reply:
Can we also test with token missing --> bad request response?

Reply:
this use case has already a test in this class